### PR TITLE
Issue #236: Add Configure Sort message to allow translation

### PR DESCRIPTION
**Pull request by RubenEtendo**
EPL-1085

### CODE REVIEW

File: modules_core/org.openbravo.userinterface.smartclient/web/org.openbravo.userinterface.smartclient/js/ob-smartclient-labels.js [hunk 1]
```diff
@@ -201,6 +201,9 @@
   isc.ListGrid.addProperties({
     sortFieldAscendingText: getLabel('OBUISC_ListGrid.sortFieldAscendingText')
   });
+  isc.ListGrid.addProperties({
+      configureSortText: getLabel('OBUISC_ListGrid.configureSortText')
+    });
```
Comment (sebastianbarrozo) [inline modules_core/org.openbravo.userinterface.smartclient/web/org.openbravo.userinterface.smartclient/js/ob-smartclient-labels.js:206]: Bad formatting

File: sonar-project.properties [hunk 2]
```diff
@@ -1,3 +1,3 @@
-sonar.projectKey=etendosoftware_etendo_core_AYOKvQCAuJ79WHyLB97g
```
Comment (sebastianbarrozo) [inline sonar-project.properties:1]: Is needed to change sonar project key? This will cause sonar history reset.

**Review by sebastianbarrozo — CHANGES_REQUESTED** [review]

File: sonar-project.properties (hunk 2 above)
Comment (RubenEtendo) [reply sonar-project.properties:1]: My bad, I didn't notice that change getting uploaded

**Review by RomanMagnoli — APPROVED** [review]

File: modules_core/org.openbravo.userinterface.smartclient/web/org.openbravo.userinterface.smartclient/js/ob-smartclient-labels.js (hunk 1 above)
Comment (RubenEtendo) [reply modules_core/org.openbravo.userinterface.smartclient/web/org.openbravo.userinterface.smartclient/js/ob-smartclient-labels.js]: Done!

File: sonar-project.properties (hunk 2 above)
Comment (RubenEtendo) [reply sonar-project.properties]: Done!

**Review by sebastianbarrozo — APPROVED** [review]

**Pull request merged**